tests/test_chain.py

```diff
@@ -303,3 +304,56 @@ def test_deposit_replacement(lianad, bitcoind):
     addr = lianad.rpc.getnewaddress()["address"]
     bitcoind.rpc.sendtoaddress(addr, 2)
     wait_for(lambda: len(lianad.rpc.listcoins()["coins"]) == 1)
+
+def test_rescan_and_recovery(lianad, bitcoind):
+    """Test user recovery flow"""
+    # Get initial_tip to use for rescan later
+    initial_tip = bitcoind.rpc.getblockheader(bitcoind.rpc.getbestblockhash())
+
+    # Start by getting a few coins
+    destinations = {
+        lianad.rpc.getnewaddress()["address"]: 0.5,
+    }
+    txid = bitcoind.rpc.sendmany("", destinations)
```

Comment:
nit: if you only have a single destination, no need to `sendmany` you can just `sendtoaddress`.